src/test/java/com/exadel/demo/controller/UserControllerTest.java

```diff
@@ -0,0 +1,182 @@
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.ArrayList;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = {UserController.class})
+@ExtendWith(SpringExtension.class)
+class UserControllerTest {
+
+    @Autowired
+    private UserController userController;
+
+    @MockBean
+    private UserService userService;
+
+    @MockBean
+    private Logger logger;
+
+    @Test
+    void testGetAllUsers() throws Exception {
+        doNothing().when(this.logger).info((String) any());
+        when(this.userService.getAllUsers()).thenReturn(new ArrayList<>());
+        MockHttpServletRequestBuilder requestBuilder = MockMvcRequestBuilders.get("/user/getAll");
+        MockMvcBuilders.standaloneSetup(this.userController)
+                .build()
+                .perform(requestBuilder)
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType("application/json"))
+                .andExpect(MockMvcResultMatchers.content().string("[]"));
+    }
+
+    @Test
+    void testGetAllUsers2() throws Exception {
+        doNothing().when(this.logger).info((String) any());
+        when(this.userService.getAllUsers()).thenReturn(new ArrayList<>());
+        MockHttpServletRequestBuilder getResult = MockMvcRequestBuilders.get("/user/getAll");
+        MockMvcBuilders.standaloneSetup(this.userController)
+                .build()
+                .perform(getResult)
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType("application/json"))
+                .andExpect(MockMvcResultMatchers.content().string("[]"));
+    }
+
+    @Test
+    void testGetUserById() throws Exception {
+        doNothing().when(this.logger).info((String) any());
+        when(this.userService.getUser((Long) any())).thenReturn(new UserDto());
+        MockHttpServletRequestBuilder getResult = MockMvcRequestBuilders.get("/user/get/*");
+        MockHttpServletRequestBuilder requestBuilder = getResult.param("id", String.valueOf(1L));
+        MockMvcBuilders.standaloneSetup(this.userController)
+                .build()
+                .perform(requestBuilder)
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType("application/json"))
+                .andExpect(MockMvcResultMatchers.content().string("{\"firstName\":null,\"lastName\":null,\"email\":null,\"roleDto\":null}"));
```

Comment:
I think you can create variable for "{\"firstName\":null,\"lastName\":null,\"email\":null,\"roleDto\":null}"